doc/user/data/sql_funcs.yml

```diff
@@ -755,6 +755,7 @@
     Functions whose primary purpose is to facilitate compatibility with PostgreSQL tools.
     These functions may have suboptimal performance characteristics.
   functions:
+<<<<<<< HEAD
```

Comment:
I think you may have left some git conflicts in here.

Reply:
Yep. Mucked up a rebase. I just fixed.